internal/hud/webview/view.go

```diff
@@ -78,6 +78,7 @@ type Resource struct {
 	PendingBuildSince  time.Time
 
 	Endpoints []string
+	PodIDs    []string
```

Comment:
this should probably be a `[]k8s.PodID`? the serializer will do the right thing

Reply:
Ok, so `PodIDs` is type `[]k8s.PodID`, but `Endpoints` is OK as `[]string`?

Reply:
ya, though i would be fine with creating an Endpoint typedef